src/silverlabnwb/nwb_file.py

```diff
@@ -125,7 +125,11 @@ def create_from_metadata(self, metadata_file, user=None, session_id=None):
         try:
             start_time = sessions[user]['start_time']
         except KeyError:
-            raise ValueError("Start time for session not found!")
+            raise ValueError("Start time for session not found! "
+                             "Please add a `start_time` parameter to sessions for user {}, "
+                             "e.g. '10 May 2020 12:34:56.789103'"
```

Comment:
Maybe clarify that the time should be right after the key:
```suggestion
                             "e.g. start_time: 10 May 2020 12:34:56"
```
(also removing the decimals so as not to imply you need all the accuracy - the date/time parsing is quite flexible, I think)